pkg/openstack/routes.go

```diff
@@ -264,6 +268,70 @@ func updateAllowedAddressPairs(network *gophercloud.ServiceClient, port *ports.P
 	return unwinder, nil
 }
 
+func updateSecurityGroup(network *gophercloud.ServiceClient, port *ports.Port, newPairs []ports.AddressPair) (func(), error) {
+	segName := "occm-nodes-sg"
+	onFailure := newCaller()
+	var sgId string
+	var err error
+	sgId, err = secgroups.IDFromName(network, segName)
+	if err != nil {
+		if strings.Contains(err.Error(), "Unable to find") {
+			res := groups.Create(network, groups.CreateOpts{Name: segName})
+			if res.Err != nil {
+				return nil, res.Err
+			}
+			sgGroup, err := res.Extract()
+			if err != nil {
+				return nil, err
+			}
+			sgId = sgGroup.ID
+			ruleRes := rules.Create(network, rules.CreateOpts{Direction: rules.DirIngress, SecGroupID: sgId, RemoteGroupID: sgId})
```

Comment:
We've learned that using `RemoteGroupID` causes performance issues in Neutron. I'd rather opt to use CIDRs instead.